src/game/client/draw/w3dmodeldraw.h

```diff
@@ -395,20 +409,20 @@ class W3DModelDraw : public DrawModule, public ObjectDrawInterface
     }
 
 private:
-    ModelConditionInfo *m_curState;
-    ModelConditionInfo *m_nextState;
+    const ModelConditionInfo *m_curState;
+    const ModelConditionInfo *m_nextState;
     int m_loopDuration;
     int m_hexColor;

```

Comment:
int m_indicatorColor

Reply:
We know this was the original name so lets use it.